pkg/operator/gcwatchercontroller/prometheus_client.go

```diff
@@ -37,12 +37,14 @@ func newPrometheusClient(ctx context.Context, configMapClient corev1client.Confi
 	roots := x509.NewCertPool()
 	roots.AppendCertsFromPEM(bundlePEM)
 
+	dialer := &net.Dialer{
```

Comment:
nit: Why do you need a new variable here? It is used only once and for read-only op

Reply:
this was hitting my IDE linter and I forgot why, and I am not seem to able to trigger it again (maybe due to an update?). I have reverthed this change